src/libNode/PoWProcessing.cpp

```diff
@@ -170,6 +170,9 @@ bool Node::StartPoW(const uint64_t& block_num, uint8_t ds_difficulty,
                                   fixedDSBlockDistributionDelayTime +
                                   extraWaitTime)) == cv_status::timeout) {
         lock_guard<mutex> g(m_mutexDSBlock);
+
+        POW::GetInstance().StopMining();
```

Comment:
@ShengguangXiao is this move ok? It looks like it's a bug fix (stopmining would not have been called previously if DS block was processed during time out).